draft-ietf-quic-transport.md

```diff
@@ -1296,6 +1291,22 @@ connection ID, it should expect packets on the new connection ID to start at 18.
 A packet with the new connection ID and a packet number of 17 is discarded as
 being in error.
 
+#### Packet Number Gap
+
+In order to avoid linkage, the packet number gap MUST be externally
+indistinguishable from random. The packet number gap for a connection
+ID with sequence number is computed by encoding the sequence number
+as a 32-bit integer in big-endian format, and then computing:
+
+~~~
+Gap = HKDF-Expand-Label(packet_number_secret,
+                        "QUIC packet sequence gap", sequence, 4)
```

Comment:
@janaiyengar, is what you want here:

"The TLS-QUIC API must expose a function QUIC-Packet-Gap(sequence) -> Gap which returns a 32-bit integer indicating the packet gap for a given sequence number"?